operator/cmd/syncclusterconfig/sync.go

```diff
@@ -98,14 +95,6 @@ If present and not empty, the $%s environment variable will be set as the cluste
 				return err
 			}
 
-			// NB: remove must be an empty slice NOT nil.
```

Comment:
Expand the diff here. `syncer.Sync` is called right before this. I think we made it through will no issues because `clusterConfig` was mutated so heavily?